popup.js

```diff
@@ -13,38 +13,60 @@ var developerTools = {
 			chrome.tabs.update(tabs[0].id, {url: tabUrl.origin + tabUrl.pathname + '?' + params.toString()});
 		});
 
+	},
+	bustCache: function(cacheBuster) {
```

Comment:
If possible, I would love to reuse the `debugReload` rather than having a separate, but similar function here. Why don't we pass a second argument into the `debugReload` function, the `debugParam` and the `debugParamValue`. We can pass in `True` for the hsDebug and jQueryFooter and and you can use your random number generation for `cacheBuster`. Thoughts?

Reply:
I implemented it a bit different but still merged the functions feel free to change if you think you have a better method. I wasn't 100% sure I understood your thought on this.

Reply:
This was my thought:
```
debugReload: function(debugParam, debugParamValue) {
	chrome.tabs.query({active: true, currentWindow: true}, function(tabs) {
		var tabUrl = new URL(tabs[0].url);
		var params = new URLSearchParams(tabUrl.search);
		if (params.has(debugParam)) {
			params.delete(debugParam);
		} else {
			params.append(debugParam, debugParamValue);
		}
		chrome.tabs.update(tabs[0].id, {url: tabUrl.origin + tabUrl.pathname + '?' + params.toString()});
	});

}

...


$('.js-click--debug,.js-click--move-jquery-to-footer,.js-click--bust-cache').click(function () {
	if ($(this).attr('id') == 'cacheBuster'){
		var debugParamValue = Math.floor(Math.random() * 9999) + 1;
	} else {
		var debugParamValue = 'True';
	}
	developerTools.debugReload($(this).attr('id'), debugParamValue);
});
```

What do you think?

Reply:
My only issue with it is just that I think with the cache buster it's more useful if when you hit the button a second time it just generates a new number, doesn't remove the parameter.

I find when dev-ing a site on HS I end up having to add a query parameter and keep updating it to keep seeing the uncached version of the page, so I would end up doing changing that number a lot. then wipe it out when I want to see if HS's cache has cleared.

Reply:
Ahhhhhh I understand now

Reply:
does it make sense to go with the js I put up above or did you want it modified to follow the parameter method you had above. I don't know that it's necessary

Reply:
Sorry for the confession, I think for now its not a big deal but nice to re-use for the most sustainable setup. 

Last suggestion, otherwise LGTM for now
```
if (debugParam == "cacheBuster") {
	var randomNum = Math.floor(Math.random() * 9999) + 1;
	params.set("cacheBuster",randomNum);
} else if (params.has(debugParam)) {
	params.delete(debugParam);
} else {
	params.append(debugParam, "True");
}
```

Reply:
updated and it tests fine. Will merge